### PR TITLE
PORTALS-2101

### PR DESCRIPTION
**Pull request by yjyoo773**

### CODE REVIEW

File: src/Navbar.tsx [hunk 1]
```diff
@@ -9,6 +9,9 @@ import NavLink from 'portal-components/NavLink'
 import NavUserLink from './portal-components/NavUserLink'
 import { GenericRoute } from 'types/portal-config'
 import Button from 'react-bootstrap/esm/Button'
+import IconButton from '@material-ui/core/IconButton'
```
Comment (nickgros) [inline src/Navbar.tsx:12]: @yjyoo773 from the issue comments, I thought we decided to convert this to react-bootstrap. Is there a reason you decided to continue using Material-UI?

File: src/AppInitializer.tsx [hunk 2]
```diff
@@ -114,7 +114,8 @@ class AppInitializer extends React.Component<Props, AppInitializerState> {
     })
   }
 
-  handleCloseLoginDialog = () => {
+  handleCloseLoginDialog = (event, reason) => {
+    if(reason && reason === "backdropClick") return
```
Comment (nickgros) [inline src/AppInitializer.tsx:118]: Is this still necessary now that we have `backdrop='static'`?

Comment (yjyoo773) [reply src/AppInitializer.tsx:118]: No it isn't. I'll remove this

File: src/Navbar.tsx [hunk 3]
```diff
@@ -192,15 +192,15 @@ class Navbar extends React.Component<any, State> {
...
-                  >
-                    <SynapseComponents.Login
-                      sessionCallback={() => getSession()}
-                    />
-                  </Dialog>
+                  <Modal 
+                    animation={false}
+                    backdrop={'static'}
+                    onHide={handleCloseLoginDialog} 
+                    style={{'marginTop':'5%'}}
```
Comment (nickgros) [inline src/Navbar.tsx:199]: Would prefer this to be a number (px) in case the width of this component changes in the future.

**Review by nickgros — CHANGES_REQUESTED** [review]
Small questions/changes requested but overall looks good!

**Review by nickgros — APPROVED** [review]

**Pull request merged**